draft-ietf-taps-arch.md

```diff
@@ -223,7 +223,10 @@ The Transport Services API represents data as messages, so that it more closely
 * the ability to control reliability, which messages to retransmit when there is packet loss, and how best to make use of the data that arrived;
 * the ability to automatically assign messages and connections to underlying transport connections to utilize multi-streaming and pooled connections.
 
-Allowing applications to interact with messages is backwards-compatible with existing protocols and APIs because it does not change the wire format of any protocol. Instead, it gives the protocol stack additional information to allow it to make better use of modern transport services, while simplifying the application's role in parsing data. For protocols which natively use a streaming abstraction, framers ({{datatransfer}}) bridge the gap between the two abstractions.
+This can have several benefits:
```

Comment:
This is strange now. I suggest to look at the rich diff and show text above this PR as well: just above, there is already a bullet list of possible benefits.

Reply:
You're correct - fixed one part to mess up another, I think I see how it should have been, will return to that